src/components/user-card/UserCard.js

```diff
@@ -15,3 +16,76 @@ import React from 'react';
 //  этот div будет содержать:
 //  - блок h4 с классом card-title, он должен вывести имя и фамилию (first_name, last_name)
 //  - блок div c классом "card-text", содержащий 2 div блока для поля email и поля address
+
+export const UserCard = (props) => {
+const { usersList } = props;
+
+if (!usersList) return null;
+
+const { first_name, last_name, email, address, _links } = usersList;
```

Comment:
This code doesn't make any sense cause you are passing an array as usersList
array doesn't have such properties as first_name, last_name, email, address, _links
that's why bellow you are still using usersList[0]

Should be like that:
component gots`user` in props 
user object has such properties `first_name, last_name, email, address, _links`
you are using these properties from user all through jsx (where now there are `usersList[0].`)